app/controllers/users_controller.rb

```diff
@@ -5,7 +5,7 @@ def index
   end
 
   def show
-    @user = User.where(id: params[:id])
+    @user = User.find(params[:id])
     @posts = Post.where(id: params[:id])
```

Comment:
- [ ] <b>REQUIRED:</b> You can simply get the posts that belong to the user by : `@posts = @users.posts` and don't forget to fix N+1 problem which would be like this: `@posts = @users.posts.includes(:comments, :likes)`